ugvc/mrd/srsnv_plotting_utils.py

```diff
@@ -63,7 +82,7 @@ def create_data_for_report(
         predictions_df[f"XGB_prob_{label}"] = probs[:, label]
         predictions_df[f"XGB_qual_{label}"] = quals[:, label]
         predictions_df[f"XGB_prediction_{label}"] = predictions[:, label]
-    df = pd.concat([X, predictions_df.drop(["label"], axis="columns", inplace=False)], axis=1)
+    df = pd.concat([X, predictions_df], axis=1)
```

Comment:
This is kind of an awkward way to do that. If you are just assigning columns this should do:

`df = X.assign(**{f"XGB_prob_{label}": probs[:, label] for label in labels}).assign(**{f"XGB_qual_{label}": quals[:, label] for label in labels}).assign(**{f"XGB_prediction_{label}": predictions[:, label] for label in labels})`